tools/sugarbin_shell_tier.py

```
@@ -0,0 +1,339 @@
+        _write_json(
+            body_path,
+            _body(
+                contract,
+                commit=commit,
+                status="running",
+                reason="command-started",
+            ),
+        )
+        started = time.monotonic()
+        try:
+            completed = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)